### PR TITLE
Split Landal.* from Wyndham.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Wyndham.xml [hunk 1]
```diff
@@ -126,26 +123,13 @@ Fetch error: http://www.wyndhamrentals.com/ => https://www.wyndhamrentals.com/:
...
 
 	<rule from="^http://www\.landacampings\.(be|nl)/(css|favicons|img)/"
 		to="https://www.landalcampings.$1/$2/"/>
 
-	<rule from="^http://secure\.landal\.com/"
-		to="https://secure.landal.com/"/>
-
 	<!--	redirects as so	-->
 	<rule from="^http://(?:www\.)?landalgreenparks\.com/"
 		to="https://www.landal.com/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Wyndham.xml]: What about that one?

Comment (cschanaj) [reply src/chrome/content/rules/Wyndham.xml]: I'm sorry. Can you be more specific?

Comment (J0WI) [reply src/chrome/content/rules/Wyndham.xml]: The `landalgreenparks.com` hosts are redirected to `landal.com`. If you plan to keep this rule it should be in the `Landal.xml` file.

**Comment by cschanaj** [comment]
@J0WI I have moved `landalgreenparks.com` to `Landal.xml`. Thanks!!

**Pull request merged**